helpers/mathjax.js

```diff
@@ -47,6 +47,7 @@ class HtmlBlockMathRenderer {
 
 		const mathJaxConfig = {
 			deferTypeset: true,
+			enableMML3Support: context.enableMML3Support,
```

Comment:
This doesn't exist yet, but I'll be creating a feature flag in the LMS to provide it. I'm uncomfortable enabling anything experimental without at least having the ability to turn it off quickly.